src/TabMenu/spec.js

```diff
@@ -28,6 +28,30 @@ export default [{
    default: '<code>0px</code>',
    description: 'Set spacing between <code>TabMenuButton</code> components.',
    responsive: false
+}, {
+   name: 'buttonTextColor',
+   type: 'String',
+   default: '<code>#888</code>',
+   description: 'Text <code>color</code> of the <code>TabMenuButton</code> when <code>isActive</code> is set to <code>false</code>',
+   responsive: false
+}, {
+   name: 'buttonTextFontFamily',
+   type: 'OIO Responsive String',
+   default: '-',
+   description: 'Set <code>fontFamily</code> for all <code>TabMenuButton</code> components.',
+   responsive: true
+}, {
+   name: 'buttonTextSize',
```

Comment:
If the size is a bit bigger, the text ends up being taller than the actual component, but there doesn't seem to be a way to adjust the height of the `TabMenu`?

![Screen Shot 2019-11-09 at 4 54 09 PM](https://user-images.githubusercontent.com/324828/68535525-aa9d8a00-0311-11ea-87fb-8ec2bd0ae48b.png)

Reply:
Yeah, I assume that we will add a way to adjust the button's height manually, but I am still thinking about the best way to do so. There is a possibility that the `buttonSize` prop will be removed altogether, but I am still thinking about it.